mkosi/resources/mkosi.md

```diff
@@ -375,7 +375,7 @@ boolean argument: either `1`, `yes`, or `true` to enable, or `0`, `no`,
 
 : The distribution to install in the image. Takes one of the following
   arguments: `fedora`, `debian`, `ubuntu`, `arch`, `opensuse`, `mageia`,
-  `centos`, `openmandriva`, `rocky`, `alma`. If not
+  `centos`, `openmandriva`, `rocky`, `alma`, `none`. If not
```

Comment:
We should document here that if the distribution is `none` that it's up to the user to populate the rootfs via a combination of base trees, skeleton trees and prepare scripts.

Reply:
Done, please check the phrasing